doc/content/getting-started/_index.md

```diff
@@ -5,7 +5,88 @@ weight: -2
+
+The messages that originates from an end device are called an Uplinks.
+
+Messages flowing in the opposite direction (originating from the Network Server and/or Application Server and sent to end devices) are called Downlinks.
+
+{{< note "Sensor networks in LoRaWAN are deployed in a star-of-stars topology." />}}
+
+### Security
+
+All LoRaWAN data is encrypted AES-128 symmetric keys. All devices have a unique AES-128 key called the "Root Key" associated with them.
+
+This root key is used to derive separate keys for the application data and network data.
+
+Application data is encrypted using one of these derived keys. This key is known only to the Application Server and the End Device.
+
+Network (settings) data is encrypted using different key(s). These keys are known only to the Network Server and the End Device.
+
+{{< note "These keys are called Session Keys since they are rotated when a device session changes. Device sessions are a much deeper topic and is omitted from this basic guide in the interest of simplicity. See the linked references for more information." />}}
+
+## {{% tts %}}
+
+{{% tts %}} is an enterprise grade LoRaWAN server (that includes both the Network Server and Application Server functions mentioned above). In addition, {{% tts %}} contains services and tools to securely manage millions of LoRaWAN devices in production.
+
+{{% tts %}} offers the following options for LoRaWAN deployments in production.
+
+- The quickest way to get started with LoRaWAN is to sign up to [{{% tts %}} Cloud]({{< relref "cloud" >}}) a fully-managed SLA-backed cloud subscription. By using {{% tts %}} Cloud, you can focus on scaling your LoRaWAN deployments while leaving the complexity of managing a production LoRaWAN Network Server to us. The Things Stack Cloud offers a [free discovery tier](https://www.thethingsindustries.com/stack/plans/) to get started and is the recommended option for most new users.
+
+- Alternatively, if you prefer to take on the responsibility of installing and maintaining {{% tts %}} in addition to managing your LoRaWAN device fleet, {{% tts %}} is available to be [installed on your own hardware or cloud infrastructure]({{< relref "setup" >}}).
+
+For simple community projects and local testing, there are a few options.
+
+- A free and limited version of {{% tts %}} called [{{% tts %}} Community Edition]{{< relref "ttn" >}} is available to [The Things Network](https://www.thethingsnetwork.org/) community for simple proof-of-concepts and community projects. This service comes with no guarantees and is entirely at the risk of the users.
+
+- For the DIY’ers, the core features of The Things Stack are [open source](https://github.com/thethingsnetwork/lorawan-stack) and is available for local testing.
+
+{{% tts %}} is developed and maintained by [The Things Industries](https://thethingsindustries.com/).
+
+Now that we've covered some basics, let's go ahead and setup your first LoRaWAN network.
```

Comment:
Seems like an unfavorable position for this (very important for us) content to be at the bottom. What about making this a separate page as well and naming it 'Get familiar with The Things Stack' or something similar, and place the content about creating a discovery network as a subsection of that page? I feel like a majority of the page would just be scrolled over